namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-incentives-dev/resources/hmpps-events.tf

```diff
@@ -0,0 +1,3 @@
+data "aws_ssm_parameter" "hmpps-domain-events-topic-arn" {
+  name = "/hmpps-domain-events-dev/topic-arn"
```

Comment:
Created with https://github.com/ministryofjustice/cloud-platform-environments/pull/11155 and https://github.com/ministryofjustice/cloud-platform-environments/pull/11171